### ISSUE TITLE
[BUG]LazyLoadingMechanism will cause custom settings to fail

### ISSUE THREAD

**Issue by dongrentianyu** [Jermolene/TiddlyWiki5#7502]
### Describe the bug

I often use cascading mechanisms to achieve custom effects, and also use external font links to customize fonts.

Recently the latest version of tidgit supports LazyLoadingMechanism settings, but I found that when I set it to `$:/core/save/lazy-all,` my custom cascade settings and my fonts all failed.

I'm wondering if it's because these settings entries aren't prefixed with the system prefix that they're not loading in properly. If so, please remind me.

### Expected behavior

Ability to load custom settings, including cascades and fonts, etc.

### Screenshots

![`BG0TJ2OS}$EI{M1B(9MU94](https://github.com/Jermolene/TiddlyWiki5/assets/109711900/7b4950ca-72f9-4c20-b038-dc50f13df253)

![ZB}0XGD58Q)NAD_8M` Z 5V](https://github.com/Jermolene/TiddlyWiki5/assets/109711900/6d96a20d-6d82-4879-a1f9-91d72d3d8217)

![image](https://github.com/Jermolene/TiddlyWiki5/assets/109711900/ee835df0-186a-41f7-b8a3-722bae1aebb7)

The following is the normal situation.

![image](https://github.com/Jermolene/TiddlyWiki5/assets/109711900/0db29b10-4a9c-4ea0-a59c-6f462b687c7f)

### TiddlyWiki Configuration

- Version [v-5.3.0-prerelease-2023-05-22]
- Saving mechanism [TidGit v0.8.0-prerelease9]
- Plugins installed [It can be found in the following website. ]

https://dongrentianyu.github.io/WENAI

**Comment by pmario** [comment]
>... tidgit supports LazyLoadingMechanism settings

I think that's a tidgit problem and should be discussed there. Did it work with TW v5.2.7?

**Comment by dongrentianyu** [comment]
![image](https://github.com/Jermolene/TiddlyWiki5/assets/109711900/d238ed39-b65f-4f95-a79c-ea37920e3772)

![image](https://github.com/Jermolene/TiddlyWiki5/assets/109711900/77081773-aa88-474d-bb7b-91dde2b2cdec)

I am using version 5.2.5, if needed I can try again with version 5.2.7 to test it. I don't think it's a tidgit problem though.

**Comment by pmario** [comment]
hmm. Seems to work with the older version, if the first screenshot is 5.2.5 ... So more investigation will be needed. ... I do not have time for that atm. So someone else may be able to jump in. @linonetwo ??

**Comment by dongrentianyu** [comment]
The screenshot version at the beginning was the 5.3.0 pre-release version, followed by the 5.2.5 version, but I don't feel like I should have to test any other versions. At the moment, he estimated that there is no time to fix it, only delay.

**Comment by Jermolene** [comment]
Hi @dongrentianyu it sounds like the problem is indeed that the template `$:/core/save/lazy-all` affects all tiddlers that are not system tiddlers and not JavaScript tiddlers. Thus your tiddlers such as `config-time-recording-template` etc are all being loaded lazily.

Lazy loading is only intended to be applied to "content tiddlers", and will inevitably cause problems if applied to configuration tiddlers.

**Comment by valpackett** [comment]
It would be nice if there was an "official" method of determining what is a content tiddler..

For now in TiddlyPWA I'm using 'starts with `$:` or has any tag that starts with `$:`' as the "configuration-ness" test, it seems to be working well (modulo #7686 hah) but it would be nice if determining that was "not my problem" :)

**Comment by linonetwo** [comment]
What will do to content tiddler?

1. not loaded if lazy-load enabled

anything else?

Maybe just need a list like StoryList as config for them.

**Comment by Jermolene** [comment]
> It would be nice if there was an "official" method of determining what is a content tiddler..

@valpackett you are correct that the distinction between content tiddlers and "plumbing" tiddlers is that the latter should be marked as system tiddlers. The core embodies this idea in the way that most lists omit system tiddlers by default. System tiddlers are defined as having the prefix `$:/`. The core does define a `wiki.isSystemTiddler(title)` method to encapsulate the test.

**Comment by pmario** [comment]
@dongrentianyu ... As discussed here. If you use `$:/core/save/lazy-all` all tiddlers without the `$:/` prefix are lazy. So the text-field will **not** be loaded. 

So you should prefix your configuration tiddlers to be [System Tiddlers](https://tiddlywiki.com/#SystemTiddlers)
